README.md

```diff
@@ -107,27 +107,30 @@ You can check out [the API documentation](https://docs.meilisearch.com/reference
 
 Go checkout [examples](./examples)!
 
+In this section, the examples contain the [`await` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await). This operator is used to wait for a `Promise`.<br>
+Despite it is not detailled in the examples above, it can only be used inside an `async` function.
+
 ### Indexes
 
 #### Create an index <!-- omit in toc -->
 
 ```javascript
 // Create an index
-meili.createIndex({ uid: 'books' }) // if your index does not exist
+const index = await client.createIndex({ uid: 'books' })
```

Comment:
I did not really understand before you explained it (in the comment above) the `createIndex` method didn't return an object index. I thought it did because of the `createIndex` methods in the other SDKs did.